src/Resistor/Resistor.ts

```diff
@@ -33,7 +33,7 @@ class Resistor {
     for (let i = 0; i < (colors.length - 1); ++i) {
       const currentColor = colors[i];
       if (currentColor.value === undefined) {
-        throw new Error('Invalid value color');
+        return 'Invalid';

```

Comment:
Not a fan of special strings like this.  I'd say return `-1` or `null` if possible.  It should be up to the UI to determine what string to show.

It also seems like normally you're returning a number, so it's weird to also return a string down this path.

Reply:
Changed to -1